common/src/mbus_api/message_bus/v0.rs

```diff
@@ -228,6 +229,24 @@ pub trait MessageBusTrait: Sized {
         Ok(())
     }
 
+    /// set volume replica count
```

Comment:
Incorrect comment - this is the publish call

Reply:
dohh